test/libsolidity/syntaxTests/viewPureChecker/eof/inline_assembly_instructions_disallowed_pure.sol

```diff
@@ -0,0 +1,68 @@
+contract C {
+    function f() public pure {
+        assembly {
+            pop(sload(0))
+            sstore(0, 1)
+            pop(address())
+            pop(balance(0))
+            pop(caller())
+            pop(callvalue())
+            pop(extcall(0, 1, 2, 3))
+            pop(extstaticcall(0, 1, 2))
+            pop(extdelegatecall(0, 1, 2))
+            log0(0, 1)
+            log1(0, 1, 2)
+            log2(0, 1, 2, 3)
+            log3(0, 1, 2, 3, 4)
+            log4(0, 1, 2, 3, 4, 5)
+            pop(origin())
+            pop(gasprice())
+            pop(blockhash(0))
+            pop(coinbase())
+            pop(timestamp())
+            pop(number())
+            pop(gaslimit())
+            pop(blobhash(0))
+            pop(blobbasefee())
+            pop(tload(0))
+            pop(selfbalance())
+            pop(chainid())
+            pop(basefee())
+            pop(prevrandao())
```

Comment:
`tload` and `tstore` missing.

`tstore` also in `eof/inline_assembly_instructions_allowed.sol`